riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/DefaultRiptideRegistrar.java

```diff
@@ -47,20 +48,13 @@
 import java.net.URI;
 import java.util.List;
 import java.util.Optional;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-import java.util.concurrent.SynchronousQueue;
-import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.*;
 import java.util.function.Predicate;
 
 import static java.util.stream.Collectors.toCollection;
 import static org.springframework.beans.factory.support.BeanDefinitionBuilder.genericBeanDefinition;
 import static org.zalando.riptide.spring.Dependencies.ifPresent;
-import static org.zalando.riptide.spring.Registry.generateBeanName;
-import static org.zalando.riptide.spring.Registry.list;
-import static org.zalando.riptide.spring.Registry.ref;
+import static org.zalando.riptide.spring.Registry.*;
```

Comment:
On purpose or just new IDE? ;-)

Reply:
Not on purpose. I didn't succeed in transferring my settings yet... :cry: